p4/include/tps_consts.p4

```diff
@@ -13,12 +13,30 @@
 # limitations under the License.
 */
 /* -*- P4_16 -*- */
-#define MAX_HOPS 9
+#define TABLE_SIZE 1024
+#define INT_SHIM_BASE_SIZE 7
+#define INT_SHIM_HOP_SIZE 1
+#define BYTES_PER_SHIM 4
+#define UDP_HDR_BYTES 8
+#define TCP_HDR_BYTES 20
+#define IPV6_HDR_BYTES 40
 
-const bit<8> TYPE_INSPECTION = 0xfd;
+const bit<8> MAX_HOPS = 0xa;
+
+const bit<4> INT_SHIM_TYPE = 0x1;
+const bit<16> INT_SHIM_DOMAIN_ID = 0x5453;
+const bit<2> INT_SHIM_NPT_UDP_FULL_WRAP = 0x2;
+const bit<4> INT_VERSION = 0x2;
+const bit<5> INT_META_LEN = 0x1;
+const bit<16> UDP_INT_SRC_PORT = 0x0;
+const bit<16> UDP_INT_DST_PORT = 0x022b;
+const bit<16> TPS_UDP_PORT = 0x216b;
 const bit<16> TYPE_IPV4 = 0x0800;
 const bit<16> TYPE_IPV6 = 0x86dd;
 const bit<8> TYPE_TCP = 0x06;
 const bit<8> TYPE_UDP = 0x11;
```

Comment:
Add comments for defines and constants. What is it used for and why was that value selected.

Perhaps more descriptive names for some of the generic names.

Reply:
I'll push in the comments soon.